### ISSUE TITLE
NullInjectorError: No provider for AnimalExaminationService! When creating an exam while importing Bruker

### ISSUE THREAD

**Issue by arthursw** [fli-iam/shanoir-ng#750]
Error when creating an exam while importing a Bruker:

```
handle-error.service.ts:48 Error: Uncaught (in promise): NullInjectorError: R3InjectorError(AppModule)[AnimalExaminationService -> AnimalExaminationService -> AnimalExaminationService]: 
  NullInjectorError: No provider for AnimalExaminationService!
NullInjectorError: R3InjectorError(AppModule)[AnimalExaminationService -> AnimalExaminationService -> AnimalExaminationService]: 
  NullInjectorError: No provider for AnimalExaminationService!
    at NullInjector.get (core.js:915)
    at R3Injector.get (core.js:11075)
    at R3Injector.get (core.js:11075)
    at R3Injector.get (core.js:11075)
    at NgModuleRef$1.get (core.js:24194)
    at Object.get (core.js:22095)
    at getOrCreateInjectable (core.js:3921)
    at Module.ɵɵdirectiveInject (core.js:13746)
    at NodeInjectorFactory.AnimalExaminationFormComponent_Factory [as factory] (animal-examination-form.component.ts:56)
    at getNodeInjectable (core.js:4029)
    at resolvePromise (zone-evergreen.js:798)
    at resolvePromise (zone-evergreen.js:750)
    at zone-evergreen.js:860
    at ZoneDelegate.invokeTask (zone-evergreen.js:399)
    at Object.onInvokeTask (core.js:27419)
    at ZoneDelegate.invokeTask (zone-evergreen.js:398)
    at Zone.runTask (zone-evergreen.js:167)
    at drainMicroTaskQueue (zone-evergreen.js:569)
    at ZoneTask.invokeTask [as invoke] (zone-evergreen.js:484)
    at invokeTask (zone-evergreen.js:1621)
```

**Comment by arthursw** [comment]
Should be just a matter of declaring the AnimalExaminationService in app.module.ts!

**Comment by arthursw** [comment]
Probably related to  #745 and fixed with #753
